modules/persistent-storage-csi-vsphere-encryption-tag-based.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-vsphere.adoc
+//
+
+:content-type: PROCEDURE
+[id="persistent-storage-csi-vsphere-encryption-tag-based_{context}"]
+= Using tag-based placement
+
+.Procedure
+
+To encrypt using tag-based placement:
+
+. In vCenter create a category for tagging datastores that will be made available to this storage class. Also, ensure that StoragePod, Datastore, and Folder are selected as Associable Entities for the created category.
```

Comment:
* Are all of these steps to be completed in vCenter? If so, I'd suggest adding a first step that is similar to: `. Open the VMware vCenter vSphere client GUI.` (wording taken from another part of this assembly)

* > StoragePod, Datastore, and Folder are selected as Associable Entities

  Should these terms be in bold formatting as GUI elements?

* [minimalism suggestion] I'm not familiar with this GUI, but my inclination is to suggest that you rephrase the last sentence to something like `Select the following *Associable Entities* for the new category: *StoragePod*, *Datastore*, and *Folder*.`